pkg/virt-launcher/virtwrap/network/common_test.go

```diff
@@ -106,18 +106,29 @@ var _ = Describe("Common Methods", func() {
 var _ = Describe("VIF", func() {
 	Context("String", func() {
 		It("returns correct string representation", func() {
-			addr, _ := netlink.ParseAddr("10.0.0.200/24")
-			mac, _ := net.ParseMAC("de:ad:00:00:be:ef")
-			gw := net.ParseIP("10.0.0.1")
-			vif := &VIF{
-				Name:      "test-vif",
-				IP:        *addr,
-				MAC:       mac,
-				Gateway:   gw,
-				Mtu:       1450,
-				TapDevice: "myTap0",
-			}
-			Expect(vif.String()).To(Equal("VIF: { Name: test-vif, IP: 10.0.0.200, Mask: ffffff00, MAC: de:ad:00:00:be:ef, Gateway: 10.0.0.1, MTU: 1450, IPAMDisabled: false, TapDevice: myTap0}"))
+			vif := createVIF()
+			Expect(vif.String()).To(Equal("VIF: { Name: test-vif, IP: 10.0.0.200, Mask: ffffff00, IPv6: <nil>, MAC: de:ad:00:00:be:ef, Gateway: 10.0.0.1, MTU: 1450, IPAMDisabled: false, TapDevice: myTap0}"))
+		})
+		It("returns correct string representation with ipv6", func() {
+			vif := createVIF()
+			ipv6Addr, _ := netlink.ParseAddr("fd10:0:2::2/120")
+			vif.IPv6 = *ipv6Addr
```

Comment:
This is not nice.
How about adjusting `createVIF` to accept the data? (IPs, mac, GW).

I would also prefer to see here the values as constants, to assure we are comparing the correct thing.

Reply:
Moving the values to be constants is not in the scope of this PR.
Regarding adjusting the `createVIF`- I think it is readable the way it is.
The `createVIF` method creates some default vif and when needed extra value can be added outside of the method.
Passing IPs, mac, GW as parameters to the methods is non needed here since only two places use the method and noth need the same values. The only difference is the ipv6 value.

Reply:
I agree that constants would be nice, but not in the scope of this PR.
Changed `createVIF` to `createDummyVIF`

Reply:
Fair enough.

Reply:
done.